Contracts/TicTacToe/src/main.sw

```diff
@@ -0,0 +1,250 @@
+        return Winners.None;
+    }
+
+
+    fn horizontal_alignement(gameID: u256) -> Winners {   
+        if (
+            get_player_position_filled(gameID, 1) == get_player_position_filled(gameID, 2) && get_player_position_filled(gameID, 2)  == get_player_position_filled(gameID, 3) && get_player_position_filled(gameID, 1) != Players.None  
+        ) {
+            return get_player_position_filled(gameID, 1);
+        }
+        elif (
+            get_player_position_filled(gameID, 4) == get_player_position_filled(gameID, 5) && get_player_position_filled(gameID, 5)  == get_player_position_filled(6) && get_player_position_filled(4) != Players.None
+        ) {
+            return get_player_position_filled(gameID, 4);
+        }
+        elif (
+            get_player_position_filled(gameID, 7) == get_player_position_filled(gameID, 8) && get_player_position_filled(gameID, 8)  == get_player_position_filled(gameID, 9) && get_player_position_filled(gameID, 7) != Players.None
+        ) {
+            return get_player_position_filled(gameID, 7);
+        }
+        return Players.None;
+    }
+    
+    fn vertical_alignement(gameID: u256) -> Winners {
+        if (
+            get_player_position_filled(gameID, 1) == get_player_position_filled(gameID, 4) && get_player_position_filled(gameID, 4)  == get_player_position_filled(gameID, 7) && get_player_position_filled(gameID, 1) != Players.None  
+        ) {
+            return get_player_position_filled(gameID, 1);
+        }
+        elif (
+            get_player_position_filled(gameID, 2) == get_player_position_filled(gameID, 5) && get_player_position_filled(gameID, 5)  == get_player_position_filled(gameID, 8) && get_player_position_filled(gameID, 2) != Players.None
+        ) {
+            return get_player_position_filled(gameID, 2);
+        }
+        elif (
+            get_player_position_filled(gameID, 3) == get_player_position_filled(gameID, 6) && get_player_position_filled(gameID, 6)  == get_player_position_filled(gameID, 9) && get_player_position_filled(gameID, 3) != Players.None
+        ) {
+            return get_player_position_filled(gameID, 3);
+        }
+        return Players.None;
+    }
+    
+    fn diagonal_alignement(gameID: u256) -> Winners {
+        if (
+            get_player_position_filled(gameID, 1) == get_player_position_filled(gameID, 5) && get_player_position_filled(gameID, 5)  == get_player_position_filled(gameID, 9) && get_player_position_filled(gameID, 1) != Players.None  
+        ) {
+            return get_player_position_filled(gameID, 1);
+        }
+        elif (
+            get_player_position_filled(gameID, 3) == get_player_position_filled(gameID, 5) && get_player_position_filled(gameID, 5)  == get_player_position_filled(gameID, 7) && get_player_position_filled(gameID, 3) != Players.None
+        ) {
+            return get_player_position_filled(gameID, 2);
+        }
+        return Player.None;
+    }
```

Comment:
this is way too difficult to read. these can probably be deleted entirely if not rewritten to get a simple calculation to figure out the position